### PR TITLE
chore: add back audience id field on config body type

### PR DESCRIPTION
**Pull request by ajwootto**
not having the audience _id causes parsing issues with older versions of WASM, but the value isn't actually used. Going forward we can just set it to an empty string to make the parser happy

### CODE REVIEW

**Review by kaushalkapasi — APPROVED** [review]

**Comment by ajwootto** [comment]
* **#856** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/js-sdks/856?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ajwootto and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/shared/bucketing-assembly-script/__tests__/types/configBody.test.ts [hunk 1]
```diff
@@ -6,6 +6,8 @@ import {
     hasConfigDataForEtag,
 } from '../bucketingImportHelper'
 import cloneDeep from 'lodash/cloneDeep'
+import immutable from 'object-path-immutable'
```
Comment (kaushalkapasi) [inline lib/shared/bucketing-assembly-script/__tests__/types/configBody.test.ts:9]: what does this library give us? is it just a quick way to generate unique + immutable configs depending on what's passed to the method?

Comment (ajwootto) [reply lib/shared/bucketing-assembly-script/__tests__/types/configBody.test.ts:9]: yeah its just a more ergonomic way to immutably modify deeply nested things in an object without having to do many levels of `...` spreading and stuff

Comment (ajwootto) [reply lib/shared/bucketing-assembly-script/__tests__/types/configBody.test.ts:9]: only using it for tests

**Review by kaushalkapasi — APPROVED** [review]

File: lib/shared/bucketing-assembly-script/__tests__/types/configBody.test.ts (hunk 1 above)
Comment (kaushalkapasi) [reply lib/shared/bucketing-assembly-script/__tests__/types/configBody.test.ts:9]: ah gotcha.. nice :) this is a much cleaner option!

**Pull request merged**